client/Dockerfile

```diff
@@ -7,12 +7,23 @@
 # See https://github.com/nodejs/docker-node/blob/main/README.md#how-to-use-this-image
 FROM node:lts
 
-# ENV NODE_ENV=production
+# The base node image sets a very verbose log level.
+#ENV NPM_CONFIG_LOGLEVEL warn
 
-COPY ["package.json", "package-lock.json*", "./"]
+COPY . .
```

Comment:
`COPY ["package.json", "package-lock.json*", "./"]` was copied as-is from an example, but I think that it's a performance optimization, in case the `npm install` fails - then you haven't waisted time copying over files, you can't use either way? :slightly_smiling_face: